### DISCUSSION TITLE
Can I use Hugo-Tufte for technical documentation ?

### DISCUSSION THREAD

**Discussion by bwklein** [slashformotion/hugo-tufte#6] [Q&A] [answered]
I am thinking about how best to apply this theme to documentation of software applications. What would a Tufte software User Manual look like?

**Comment by slashformotion** [comment, marked as answer]
@bwklein i would suggest to use [Docsy](https://themes.gohugo.io/themes/docsy/) instead of Hugo-Tufte... Docsy have been specificaly designed to make  technical documentation. 

However, you can use Hugo-Tufte to create anything you want, it would be the first doc i know about written in Tufte style ! 


It could look like this => https://rmarkdown.rstudio.com/examples/tufte-handout.pdf

**Reply by bwklein** [reply]
Yes, I am already using Docsy, but the content section is quite plain.  I was wondering if adding some Tufte style to document pages within Docsy would make sense.  Like what would I use the side notes for, etc. If Tufte made a software User manual or a Technical Reference guide for simulation software, what would it look like?

**Reply by bwklein** [reply]
For example, I'd like to reimagine docs like these with 100% more Tufte. https://support.thunderheadeng.com/docs/pathfinder/2021-2/

**Reply by slashformotion** [reply]
Check the shortcodes https://github.com/slashformotion/hugo-tufte/tree/master/layouts/shortcodes , maybe you can adapt them to fit with docsy. I am not sure if it's really usefull.

> If Tufte made a software User manual or a Technical Reference guide for simulation software, what would it look like ?

I don't really know that. I thinkl he never did and i am not the best at designing things...

**Reply by slashformotion** [reply]
By the way, I don't know if docsy is up to date on your website, this is strange: 

![](https://i.imgur.com/3CBljjh.png)

**Reply by bwklein** [reply]
It floats over everything, that's not standard in Docsy. You can also close it and the open/close state will persist between page loads.

**Reply by slashformotion** [reply]
oooh okay
